automation/Commands/browser_commands.py

```diff
@@ -127,13 +128,22 @@ def get_website(url, sleep, visit_id, webdriver,
     # Sleep after get returns
     time.sleep(sleep)
 
+    # Check if the browser crashed
+    try:
+        webdriver.current_url
+    except Exception:
```

Comment:
This seems a too generous catch. Is the exception thrown here of class Exception or are there specific exceptions that we want to catch? Maybe sentry-log the Exception so that its details show up as breadcrumbs?